[Bug]: ImportError: libGL.so.1

### FAQ check

- [X] Yes, I checked the FAQ and my question has not been addressed.

### Instrument

Other (any stage)

### What happened?

When installing Eureka! on a fresh Ubuntu install (technically WSL, but that's not relevant), I encountered an error saying `ImportError: libGL.so.1: cannot open shared object file: No such file or directory`. Thankfully I was able to quickly find [this solution](https://stackoverflow.com/a/63377623/7810951) on stackoverflow. Summarizing the recommendation here in case that page gets deleted somehow, the following fixed things for me:
```
sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6 -y
```

I'll add this to our readthedocs FAQ page for anyone that encounters this issue in the future

### Error traceback output

```
ImportError while importing test module '/home/taylor/Eureka/tests/test_lightcurve_fitting.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../miniconda3/envs/eureka_starry/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_lightcurve_fitting.py:8: in <module>
    from eureka.S5_lightcurve_fitting import models, simulations
../src/eureka/__init__.py:24: in <module>
    from . import S1_detector_processing
../src/eureka/S1_detector_processing/__init__.py:2: in <module>
    from . import s1_process
../src/eureka/S1_detector_processing/s1_process.py:7: in <module>
    from jwst.pipeline.calwebb_detector1 import Detector1Pipeline
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/jwst/pipeline/__init__.py:4: in <module>
    from .calwebb_detector1 import Detector1Pipeline
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/jwst/pipeline/calwebb_detector1.py:22: in <module>
    from ..jump import jump_step
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/jwst/jump/__init__.py:1: in <module>
    from .jump_step import JumpStep
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/jwst/jump/jump_step.py:5: in <module>
    from .jump import run_detect_jumps
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/jwst/jump/jump.py:3: in <module>
    from stcal.jump.jump import detect_jumps
../../miniconda3/envs/eureka_starry/lib/python3.9/site-packages/stcal/jump/jump.py:5: in <module>
    import cv2 as cv
E   ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

### What operating system are you using?

Ubuntu 22.04

### What version of Python are you running?

Python 3.9.7

### What Python packages do you have installed?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.gitignore
+++ b/.gitignore
@@ -146,7 +146,7 @@ dmypy.json
 niriss_cython.c
 
 # Dockerfiles
-Dockerfile
+Dockerfile*
 docker-compose.yml
 docker-setup.sh
 

--- a/docs/source/faq.rst
+++ b/docs/source/faq.rst
@@ -36,6 +36,18 @@ Be sure that you are installing (or have installed) batman-package (not batman) 
 installed the wrong package you can try pip uninstalling it, but you may just need to make a whole new environment.
 In general, we strongly recommend you closely follow the instructions on the :ref:`installation` page.
 
+Issues running starry/theano code
+'''''''''''''''''''''''''''''''''
+
+Be sure that you have installed the ``pymc3`` optional dependencies when installing Eureka! as described on the
+:ref:`installation` page. If you're getting an error that includes ``theano/tensor/elemwise.py:264: TypeError``,
+you most likely need to make sure that you have the g++ compiler installed (i.e., do ``sudo apt install g++`` if you're
+on Ubuntu). If you're getting issues stating something like numpy was compiled for a different version, then you likely
+need to make sure you're starting with a new conda environment and use the ``pymc3`` optional dependencies which set a
+more strict upper-limit on the numpy version permitted. If you're getting an error that includes ``ImportError: libGL.so.1``,
+then you either need to start with a new conda environment and make sure that you only install ``opencv-python-headless``
+and not ``opencv-python``, or you need to install libgl1 by doing ``sudo apt install libgl1``.
+
 
 .. _faq-install:
 

--- a/environment_pymc3.yml
+++ b/environment_pymc3.yml
@@ -44,7 +44,6 @@ dependencies:
         - image_registration@git+https://github.com/keflavich/image_registration@master # Need GitHub version to avoid np.float issue
         - jwst==1.11.4
         - myst-parser # Needed for documentation
-        - opencv-python<4.8 # Upper limit needed for numpy<1.22
         - opencv-python-headless<4.8 # Upper limit needed for numpy<1.22
         - pymc3
         - setuptools_scm # Needed for version number

--- a/setup.cfg
+++ b/setup.cfg
@@ -83,7 +83,6 @@ pymc3 =
     theano-pymc
     # Extra version limits
     numpy>=1.20.0,<1.22 # Upper limit needed for theano, starry, and pymc3
-    opencv-python<4.8 # Upper limit needed for numpy<1.22
     opencv-python-headless<4.8 # Upper limit needed for numpy<1.22
     scipy>=1.4.0,<1.8.0 # Lower limit needed for scipy.fft, upper limit needed for theano, starry, and pymc3
     xarray<2023.10.0 # Upper limit needed for numpy<1.22

